### ISSUE TITLE
Add new term for XY sex-determination system

### ISSUE THREAD

**Issue by enemeshazi** [Tree-of-Sex/tos-ontology#19]
### In which category is your new term request?

- [X] new_definition
- [X] existing_definition

### Preferred term label

XY sex-determination system

### Synonyms

male heterogamety
XX/XY sex-chromosome system

### Textual definition

In this system of gonochoristic organisms, an individual's sex is determined by sex chromosomes, and male is the heterogametic sex (produces gametes that differ in their sex-chromosome content). In simple cases, one pair of sex chromosomes is carried by each individual: females carry two copies of the same sex chromosome (X chromosome) and males carry one copy of the X chromosome as well as one male-specific chromosome (Y chromosome). Complex XY systems also occur, where a combination of multiple chromosome pairs deteremine individual sex.

### Suggested parent term

sex determination
    sex-determination system
        XY sex-determination system

### Attribution

0000-0002-0179-1153

### Additional context

The term is added as "XY" in the current ToS ontology table. The best matching definition I could find is http://purl.obolibrary.org/obo/GSSO_000117 , but it contains an incomplete and therefore misleading list of examples.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by anyatorgasheva** [comment]
To alighn this definition with ZW system #36 , it would be:

XY sex-determination system is a chromosomal system in gonochoristic organisms where males are the heterogametic sex, typically having two distinct sex chromosomes X and Y, with varying degree of genetic content differentiation, and females (the homogametic sex) having two copies of an X chromosome. 

And also probably with similar comment:
More than one distinct X and/or Y chromosome may be also present in the particular karyotype/genome (multiple sex chromosome system). XY sex chromosome system may also co-segregate with another sex chromosome system (ZW or XY) within the same individual/population/species (complex sex chromosome systems).

**Comment by ValenzuelaLab** [comment]
To parallel the ZW sex determination system, I would favor this modified definition:

XY sex-determination system is a chromosomal system of genotypic sex determination in gonochoristic organisms where males are the heterogametic sex, carrying two distinct sex chromosomes (labeled X and Y) that differ in genetic content (and may differ morphologically or not from one another), whereas females are the homogametic sex and carry two copies of the X chromosome.

**Comment by ASEMBER2** [comment]
I agree with this definition:

XY sex-determination system is a chromosomal system of genotypic sex determination in gonochoristic organisms where males are the heterogametic sex, carrying two distinct sex chromosomes (labeled X and Y) that differ in genetic content (and may differ morphologically or not from one another), whereas females are the homogametic sex and carry two copies of the X chromosome.

We should indeed proceed with XY and ZW now, it is a stepping stone for defining the derived systems :-)

**Comment by ASEMBER2** [comment]
To follow the modification in ZW-sex-determination system, based on a discussion, I suggest this modification:

XY sex-determination system is a chromosomal system of genotypic sex determination, where males are the heterogametic sex, carrying two distinct sex chromosomes (labeled X and Y) that differ in genetic content, whereas females are the homogametic sex and carry two copies of the X chromosome.

Explanation: "gonochoristic" excluded. Hermaphroditic nematodes may have sex chromosomes (X0 C. elegans, for example).

**Comment by enemeshazi** [comment]
> To follow the modification in ZW-sex-determination system, based on a discussion, I suggest this modification:
> 
> XY sex-determination system is a chromosomal system of genotypic sex determination, where males are the heterogametic sex, carrying two distinct sex chromosomes (labeled X and Y) that differ in genetic content, whereas females are the homogametic sex and carry two copies of the X chromosome.
> 
> Explanation: "gonochoristic" excluded. Hermaphroditic nematodes may have sex chromosomes (X0 C. elegans, for example).

I agree with this modification. 'XY sex-determination system is a chromosomal system of genotypic sex determination, where males are the heterogametic sex, carrying two distinct sex chromosomes (labeled X and Y) that differ in genetic content, whereas females are the homogametic sex and carry two copies of the X chromosome.'

**Comment by PaulYannJay** [comment]
I propose a simplification, to avoid any unecessary vocabulary:

Chromosomal system of genotypic sex determination, where males carry two distinct sex chromosomes, labeled X and Y, whereas females carry two copies of the  X chromosome.

I do not think we need anything else !

**Comment by ValenzuelaLab** [comment]
I agree with this simplified definition:

Chromosomal system of genotypic sex determination, where males carry two distinct sex chromosomes, labeled X and Y, whereas females carry two copies of the X chromosome.

**Comment by enemeshazi** [comment]
> Chromosomal system of genotypic sex determination, where males carry two distinct sex chromosomes, labeled X and Y, whereas females carry two copies of the X chromosome.

I also agree with this simplified definition.
